src/services/notifications/payment.ts

```diff
@@ -1,19 +1,17 @@
-import { getCurrentPrice } from "@services/realtime-price"
-
 import { sendNotification } from "./notification"
 
 export const getTitle = {
-  "paid-invoice": ({ usd, amount }) => `+$${usd} | ${amount} sats`,
-  "onchain_receipt": ({ usd, amount }) => `+$${usd} | ${amount} sats`,
-  "onchain_receipt_pending": ({ usd, amount }) => `pending +$${usd} | ${amount} sats`,
-  "onchain_payment": ({ amount }) => `Sent onchain payment of ${amount} sats confirmed`,
+  paid_invoice: ({ usd, amount }) => `+$${usd} | ${amount} sats`,
```

Comment:
Why are these snake case and not idiomatic `camelCase`?

Reply:
thanks for pointing this out, these keys are used in mobile app. I had to rollback the paid-invoice change.